warehouse/oidc/models/github.py

```diff
@@ -138,10 +138,18 @@ class GitHubPublisherMixin:
         "environment": _check_environment,
     }
 
+    __preverified_claims__ = {
+        "iss",
+        "iat",
+        "nbf",
+        "exp",
+        "aud",
+        "jti",
+    }
```

Comment:
Can we put `jti` in `OIDCPublisherMixin` instead of duping the whole set here? I believe those claims should be strictly optional, so putting it in the mixin instead would make the diff a bit smaller.

Reply:
Another idea here, just to minimize duplication: I think this could be:

```
__preverified_claims__ = OIDCPublisherMixin.__preverified_claims__ | {"jti"}
```